product_contract/tests/test_sale_order.py

```diff
@@ -142,7 +150,7 @@ def test_action_confirm_without_contract_creation(self):
         """It should create a contract for each contract template used in
         order_line"""
         self.sale.company_id.create_contract_at_sale_order_confirmation = False
-        self.order_line1.onchange_product()
+        self.order_line1._compute_auto_renew()
```

Comment:
Is this call necessary as it should be automatic?

Reply:
Yes it is necessary